### DISCUSSION TITLE
1.3.3.0-12

### DISCUSSION THREAD

**Discussion by bb-qq** [bb-qq/aqc111#115] [Announcements]
* Applied several changes included in the latest kernel
  * fix endianness issue in aqc111_change_mtu
  * cleanup mtu related logic
  * Fix out-of-bounds accesses in RX fixup
* Supported platforms based on version 5.10 kernels
 
Note:

* The installation will fail the first time. After that, run the following command from the SSH terminal:
   * `sudo install -m 4755 -o root -D /var/packages/aqc111/target/aqc111/spk_su /opt/sbin/spk_su`
* Selecting a proper version is needed depending on your DSM version.
  * DSM 7.2 and above
    * use packages with the suffix `_7.2`.
  * DSM 7.0 and DSM 7.1
    * use packages with the suffix `_7.1`.
  * DSM 6.2.x
    * use packages with the suffix `_6.2`.

<hr /><em>This discussion was created from the release <a href='https://github.com/bb-qq/aqc111/releases/tag/1.3.3.0-12'>1.3.3.0-12</a>.</em>

**Comment by Alpha-189** [comment]
This has stopped working on my DS920. Connects for 5 minutes and then disappears. If I restart the driver it works again briefly. Anyone else having issues?

Driver version 1.3.3.0-12
DS version: DSM 7.2-64570 Update 1

**Reply by bb-qq** [reply]
Did you check the [Troubleshooting](https://github.com/bb-qq/r8152/wiki/Troubleshooting) page?
If so, please create an issue and fill in the template.

**Comment by mati188** [comment]
Hi, looks like in dsm 7.3 drivers working particulary ... DSM show only 1000Mbit conneciton on qnap usb UC5G1T  card. Is it possibility to run it somehow with full speed in 7.3 sw ?

**Reply by bb-qq** [reply]
I don't think it's directly related to the DSM update. If you haven't changed the hardware configuration, you might want to try restarting the switch or changing the cables.
